gravitee-apim-cypress/cypress/integration/rest-api/apis/mock-policy-flow.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ADMIN_USER } from 'fixtures/fakers/users/users';
+import { ApiFakers } from 'fixtures/fakers/apis';
+import { PlanFakers } from 'fixtures/fakers/plans';
+import { createApi, deleteApi, deployApi, startApi, stopApi, updateApi } from 'commands/management/api-management-commands';
+import { createPlan, publishPlan, deletePlan } from 'commands/management/api-plan-management-commands';
+import { Api } from 'model/apis';
+import { NewPlanEntity, SecurityType } from 'model/plan';
+
+context('Create a mock policy', () => {
+  let createdApi: Api;
+  let createdPlan: NewPlanEntity;
+
+  before(() => {
+    const fakeApi: Api = ApiFakers.api();
+    const fakePlan = PlanFakers.plan({ security: SecurityType.KEY_LESS });
+    createApi(ADMIN_USER, fakeApi)
```

Comment:
i would prefer to do it as an API_PUBLISHER

Reply:
Ok, that's true, good hint!